keyboards/mechlovin/hannah65/rev1/config.h

```diff
@@ -1,5 +0,0 @@
-#pragma once
-
-#define PRODUCT_ID   0x6500
-#define DEVICE_VER   0x0001
-#define PRODUCTION   Hannah65
```

Comment:
Why remove this?

Reply:
Because I have plans for Ori CLS2 PCB and Merisi PCB.
Haus, Ori-CLS2 and Merisi PCB will be in:
`qmk_firmware/keyboards/mechlovin/hannah65/rev1`